src/types/morgan.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'morgan';
```

Comment:
Kjør gjerne: `npm i -D @types/morgan` for å fikse types for denne pakken

Reply:
og slett denne filen selvsagt

Reply:
Ah, nice :+1: 
Kjørte bare en `amend` på det